src/_includes/core-nodes/read-file-use-case.md

```diff
@@ -0,0 +1,59 @@
+## What is the Read File node in Node-RED?
+
+The Read File node in Node-RED is used to read the contents of a file from the file system. In FlowFuse Cloud, the Read File node interacts with a cloud-based storage solution, leveraging AWS S3 for file storage. The Read File node can read both string and binary buffer data, making it versatile for various integration needs within Node-RED flows.
```

Comment:
Needs expention to explain how this works on devices, maybe also link the PVC issue to explain where FlowFuse is moving to.